src/main/java/net/atomique/ksar/UI/AboutBox.java

```diff
@@ -84,17 +77,28 @@ private void initComponents() {
 
         jPanel2.setLayout(new javax.swing.BoxLayout(jPanel2, javax.swing.BoxLayout.PAGE_AXIS));
 
-        urllabel.setText("website: http://sourceforge.net/projects/ksar/");
+        String url;
+
+        url ="https://github.com/vlsi/ksar";
+        urllabel.setText(String.format("<html> Website :   <a href=\"\">%s</a></html>", url ));
+        urllabel.setCursor(new Cursor(Cursor.HAND_CURSOR));
         jPanel2.add(urllabel);
+        goWebsite(urllabel, url);
 
-        authorlabel.setText("Author: xavier cherif");
+        url ="https://github.com/vlsi/ksar/CONTRIBUTION.md";
```

Comment:
But this is about "How to contribute " .. that's also missing. 

For the moment I was more thinking on a list of contributors... e.g. [antlr4 contributors](https://github.com/antlr/antlr4/blob/master/contributors.txt). The current aboutbox just lists Xavier here as author..